infrastructure/ansible/roles/traffic_ops/templates/cdn.conf.j2

```diff
@@ -81,7 +81,9 @@
          "password": "{{ to_riak_username_password }}",
          "MaxTLSVersion": "{{ to_riak_tls_max_version }}"
 {% endif %}
-      }
+      },
+      "whitelisted_oauth_urls": {{ to_whitelisted_oauth_urls | to_json }},
+      "oauth_client_secret": "{{ to_oauth_client_secret }}"
```

Comment:
Since `oauth_client_secret` is in the `traffic_ops_golang` section of `cdn.conf`, the corresponding Ansible variable should start with `to_go_`, not just `to_`.

Reply:
Done